src/libs/StartupTimer/types.ts

```diff
@@ -0,0 +1,3 @@
+type StartupTimerStop = () => void;
```

Comment:
```ts
// types.ts
type StartupTimer = {
    stop: () => void;
};

// index.native.ts
const startupTimer: StartupTimer = {
    stop: () => {
        NativeModules.StartupTimer.stop();
    },
};

// index.ts
const startupTimer: StartupTimer = {
    stop: () => {},
};

export default startupTimer;
```

What about defining `StartupTimer` type as a object with `stop` function? It would be better and more stricter.

Reply:
Done! @fabioh8010 :)